presto-product-tests/src/test/java/com/facebook/presto/tests/CreateTableTests.java

```diff
@@ -67,56 +64,41 @@ public void shouldCreateTableAsEmptySelect()
         }
     }
 
-    @Test(groups = {CREATE_TABLE, QUARANTINE})
+    @Test(groups = {CREATE_TABLE})
```

Comment:
Those tests duplicates coverage of the integration smoketests and they test almost the same code path. Or speaking more preciesely, entry point to tested code is different, but code path for tested code is the same.

Am I right? If so, I would drop those tests.